api/src/docker/openscad/openscad.js

```diff
@@ -45,11 +45,11 @@ const render = async (req, _context, callback) => {
     console.log(image, 'encoded image')
     const response = {
       statusCode: 200,
-      headers: {
-        'content-type': 'image/png',
-      },
-      body: image,
-      isBase64Encoded: true,
+      body: JSON.stringify({
```

Comment:
Instead of sending down a binary type that's an image, I'm just using json with base64 image in the json, otherwise I can't really send down the log text